upload/catalog/model/catalog/product.php

```diff
@@ -536,4 +536,13 @@ public function getTotalProductSpecials() {
 			return 0;
 		}
 	}
+
+	public function checkProductCategory($product_id, $category_ids) {
+
```

Comment:
Remove empty space. No models have spaced areas between the public function and the next line.